_announcements/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Announcements

```

Comment:
I'm assuming these entries are all placeholders for future content, but I wanted to mention that the "July 1, 2019 Announcements" link, in https://klahanie.github.io/klahanie//announcements/index.html, doesn't seem to be working.